src/yearpicker/YearPickerTop.js

```diff
@@ -19,12 +21,14 @@ export default class YearPickerTop extends React.Component {
 
   nextDate = () => this.props.changeYear(this.getYear() + 10)
 
-  getLocals() {
+  getLocals(prevIconClassName, nextIconClassName) {
```

Comment:
`getLocals` receives an object, this way it's broken:

[![https://gyazo.com/2d2f96578c7ba56f37fc8879fd9ba6e9](https://i.gyazo.com/2d2f96578c7ba56f37fc8879fd9ba6e9.gif)](https://gyazo.com/2d2f96578c7ba56f37fc8879fd9ba6e9)

you should destructure the variables you need like this:

```js
getLocals({ prevIconClassName, nextIconClassName }) {
```